source/components/about-us/about-us.css

```
@@ -3,54 +3,51 @@ main {
 }
 
 #introduction-grid {
-  height: 1000px;
-  margin: 100px;
+  height: 800px;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
 }
 #introduction-title-grid {
-  height: 400px;
+  height: 200px;
   background-color: #68a1f6;
 }
 
 #introduction-title-grid-title {
   font-size: 100px;
-  line-height: 400px;
   font-family: "Sofia";
 }
 #introduction-bottom-grid {
   height: 600px;
   background: linear-gradient(110deg, #ffd753 40%, #e990d6 40%);
   display: flex;
-  justify-content: center;
+  overflow: scroll;
 }
 
 #introduction-bottom-img {
   height: 400px;
   margin-top: 100px;
-  margin-left: 20px;
+  margin-left: 60px;
   box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.5);
 }
 #introduction-bottom-text {
-  width: 300px;
-  margin-top: 150px;
+  /* width: 300px; */
+  margin-top: 100px;
   margin-left: 7vw;
+  margin-right: 4vw;
+  margin-bottom: 100px;
+  font-size: 150%;
   text-align: left;
+  line-height: 150%;
 }
 #our-team {
-  margin: 100px 100px 50px;
-  height: 400px;
+  height: 200px;
```

Same as introduction title above.
```suggestion
  line-height: 200px;
```